examples/monitors/metric-alert/payload.json

```diff
@@ -0,0 +1,28 @@
+{
+    "tag": "prod-us-west",
+    "agg_func": "sum",
+    "agg_window": "1h",
+    "lookback": "12h",
+    "rule": "api_access",
+    "stat_types": [
+        "count"
+    ],
+    "measurement": "api_access.count",
+    "source_filters": {
+        "sourcetype": "K8s"
+    },
+    "group_bys": [
+        "k8s_namespace"
+    ],
+    "anomaly_greater_than": 60,
+    "anomaly_less_than": 80,
+    "value_greater_than": 10,
+    "value_less_than": 40,
+    "mail_recipients": "lorem.ipsum@example.org",
+    "triggers": [
+        "my-slack"
+    ],
+    "trigger_template": "notify_content:\n    title: \"Edge Delta Metric Alert\"",
```

Comment:
if we are not providing a template with rich content let's leave this empty. When it's empty the default template is used which has some useful information. e.g.
![Screenshot (71)](https://user-images.githubusercontent.com/1688634/137564755-70d1d2ab-0230-49fa-81cc-62b4eaf753ea.png)

Reply:
Got it, changed the payload in cc2d024